aim/db/models.py

```diff
@@ -162,3 +162,120 @@ def to_attr(self, session):
                     else 'consumed_contract_names')
             res_attr.setdefault(attr, []).append(c.name)
         return res_attr
+
+
+class Filter(model_base.Base, model_base.HasAimId,
+             model_base.HasName, model_base.HasDisplayName,
+             model_base.HasTenantName, model_base.AttributeMixin):
+    """DB model for Filter."""
+
+    __tablename__ = 'aim_filters'
+    __table_args__ = (uniq_column(__tablename__, 'tenant_name', 'name') +
+                      to_tuple(model_base.Base.__table_args__))
```

Comment:
I don't see the need yet. This is a simple parent-child relationship in the tree, and so far there hasn't been a need to create a backref to the parent from the child.